### PR TITLE
Define CDP interfaces that mapper needs

### PR DESCRIPTION
**Pull request by OrKoN**
This PR makes it explicit which CDP interfaces
the mapper implementation expects.

### CODE REVIEW

File: src/bidiMapper/commandProcessor.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ import {
   Script,
   Session,
 } from './domains/protocol/bidiProtocolTypes';
-import {CdpConnection} from '../cdp';
+import {CDPConnection} from './cdp';
```
Comment (sadym-chromium) [inline src/bidiMapper/commandProcessor.ts:26]: go/tsstyle#identifiers-abbreviations

File: src/bidiMapper/domains/context/browsingContextProcessor.ts [hunk 2]
```diff
@@ -328,10 +328,12 @@ export class BrowsingContextProcessor {
   }
 
   async process_cdp_sendCommand(params: CDP.SendCommandParams) {
-    const sendCdpCommandResult = await this.#cdpConnection.sendCommand(
-      params.cdpMethod,
-      params.cdpParams,
-      params.cdpSession ?? null
+    const client = params.cdpSession
```
Comment (thiagowfx) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:331]:
```suggestion
    const client = params.cdpSession;
```

File: src/bidiMapper/domains/context/browsingContextProcessor.ts [hunk 3]
```diff
@@ -328,10 +328,12 @@ export class BrowsingContextProcessor {
   }
 
   async process_cdp_sendCommand(params: CDP.SendCommandParams) {
-    const sendCdpCommandResult = await this.#cdpConnection.sendCommand(
-      params.cdpMethod,
-      params.cdpParams,
-      params.cdpSession ?? null
+    const client = params.cdpSession;
```
Comment (OrKoN) [inline src/bidiMapper/domains/context/browsingContextProcessor.ts:331]:
```suggestion
    const client = params.cdpSession
```

File: src/bidiMapper/domains/context/browsingContextProcessor.ts (hunk 2 above)
Comment (OrKoN) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:331]: suggestion does not work :) it's a ternary operator.

Comment (thiagowfx) [reply src/bidiMapper/domains/context/browsingContextProcessor.ts:331]: I'm sorry, the diff confused me ><

**Review by thiagowfx — APPROVED** [review]
Please wait for @sadym-chromium's approval on this one

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**